### PR TITLE
fix(ui): Fixed-Missing given name and last name for user search dialog

### PR DESCRIPTION
**Pull request by swapnaasg**
closes eclipse/sw360#285

Signed-off-by: Asgaonkar <swapna.asgaonkar.ext@siemens.com>

### CODE REVIEW

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayUserEdit.java [hunk 1]
```diff
@@ -104,17 +105,23 @@ public int doStartTag() throws JspException {
                 emailList.add(email);
                 if (user != null) {
                     userList.add(user.getFullname());
+                    String userDetails = user.getGivenname()+":"+user.getLastname()+":"+email+":"+user.getDepartment()+":"+user.getFullname();
+					userDetailsList.add(userDetails);
```
Comment (maxhbr) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayUserEdit.java]: code formatting: we use spaces for indentation

**Comment by imaykay** [comment]
Is it possible, that this feature branch is not in the siemens fork?

**Comment by maxhbr** [comment]
@imaykay I think the branch is in [https://github.com/swapnaasg/sw360](https://github.com/swapnaasg/sw360/tree/swapnaasg/%23285/Missing-given-name-and-last-name-for-user-search-dialog)

You can always check out the state after merging the PR 463 via:
```
prNr=463
branch="testpr/origin/$prNr"
git fetch origin "+refs/pull/$prNr/merge:"
git checkout -b "$branch" -qf FETCH_HEAD
```
(see: https://github.com/maxhbr/myconfig/blob/0a0e0762337329a11fbbe80577642075db2842f0/dotfiles/git/.gitconfig#L124-L135 )

**Comment by nutanv1contr** [comment]
For me the test result looks like.
image

**Comment by mcjaeger** [comment]
please feel free to fix and reopen

**Pull request closed without merging**